src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/data.html.twig

```diff
@@ -23,4 +23,6 @@
  * International Registered Trademark & Property of PrestaShop SA
  *#}
 
-{{ record[column.options.field]|raw }}
+<div class="text-{{ column.options.text_align }}">
```

Comment:
Isn't this going to alter **all** existing grids ? 🤔 this will change the markup everywhere ?

Reply:
Not sure why @jolelievre changed it on Develop on 2019-12-12?

Reply:
reverted these - most probably came from the rebase because their date is newer than the changes on the develop branch.